src/libasr/codegen/asr_to_python.cpp

```diff
@@ -573,6 +573,58 @@ class ASRToLpythonVisitor : public ASR::BaseVisitor<ASRToLpythonVisitor>
         s += "len(" + s + ")";
     }
 
+    void visit_StringItem(const ASR::StringItem_t &x) {
+        std::string r = "";
+        visit_expr(*x.m_arg);
+        r += s;
+        r += "[";
+        visit_expr(*x.m_idx);
+        r += s;
+        r += "]";
+        s = r;
+    }
+
+    void visit_StringSection(const ASR::StringSection_t &x) {
+        std::string r = "";
+        visit_expr(*x.m_arg);
+        r += s;
+        if (x.m_start) {
+            r += "[";
+            visit_expr(*x.m_start);
+            r += s;
+        }
+        if (x.m_end) {
+            r += ":";
+            visit_expr(*x.m_end);
+            r += s;
+        }
+        if (x.m_step) {
+            r += ":";
+            visit_expr(*x.m_step);
+            r += s;
+            r += "]";
+        } else {
+            r += "]";
+        }
+        s = r;
+    }
+
+    void visit_StringFormat(const ASR::StringFormat_t &x) {
```

Comment:
Does LPython represents `StringFormat` in the ASR? (If we don't support, let's implement this later? )
I think we don't properly support f-strings yet.

Reply:
Yes, I added these because there is an entry for `StringFormat`. Here: https://github.com/lcompilers/lpython/blob/7aa70843fe3d1f6dfd711d45220ca9c152eed506/src/libasr/ASR.asdl#L305

Do you think we should keep it? Thanks

Reply:
Yes, we have an entry in ASR.asdl, but LPython doesn't utilise it yet.
So, I think let's remove it for now. And handle it correctly once LPython supports f-strings.